newsletter/templates/newsletter/plugin.html

```diff
@@ -0,0 +1,15 @@
+{% load cms_tags static %}
+
+{% block body %}
+    <div>
+        {% if thanks %}
+            <p>Thank you. You have now subscribed to our newsletter.</p>
+        {% else %}
+            <form method="post">
+                {% csrf_token %}
+                {{ form.as_p }}
+                <button type="submit">Submit</button>
+            </form>
+        {% endif %}
+    </div>
+{% endblock body %}
```

Comment:
I'm not quite sure what this will impact if it's added as a plugin. Following others it would be better to remove the `block body` tags and have those in the index instead